c/driver/postgresql/postgres_copy_reader.h

```diff
@@ -1260,10 +1245,13 @@ class PostgresCopyStreamWriter {
     return NANOARROW_OK;
   }
 
+  const uint8_t* BufferData() const { return buffer_.value.data; }
```

Comment:
Not sure this is the best way to expose this

Reply:
hmm, maybe `const struct ArrowBuffer&`, which would give you size too? (If only we had `std::span` or `std::string_view`...)